go-server-server/go/default.go

```diff
@@ -214,6 +214,7 @@ func ConfigInterfaceVlanPost(w http.ResponseWriter, r *http.Request) {
         vnet_id_str = VNET_NAME_PREF + strconv.FormatUint(uint64(vnet_id), 10)
         vlan_if_pt.Set(vlan_name, map[string]string{
             "vnet_name": vnet_id_str,
+            "proxy_arp": "enabled",
```

Comment:
Would we want to make proxy_arp configurable via an attribute pushed by the rest-api caller?

Reply:
IMO, controller need not be aware of this. Proxy arping is required for all Vnet Vlans. Didn't want to make this default in swss as it is a generic handler for all Vlan types, so moved it to rest-api.